IFC4x3/Properties/t/TrackEdge_18qZEZZgH0YeAVlHW1LDUp/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TrackEdge_18qZEZZgH0YeAVlHW1LDUp" Name="TrackEdge" UniqueId="48d233a3-8ea4-408a-829f-bd180154d7b3" PrimaryDataType="IfcIdentifier" />
```

Comment:
Not wrong, maybe it would be easier to manage, they need this property for Topological aspect. I changed the name to TrackEdgeId. commit : 93759b0e3fae35830f1ea731228800dfd326c956
@czapplitec  what is you opinion on the subject?